README.md

```diff
@@ -1,3 +1,19 @@
-## Organizing a Collaborator Summit
+# Node.js Collaborator Summit
+
+The Node.js Foundation hosts regular summits for active contributors to the node.js project, this repository is being used to organize them.
+
+## Cost
+Free!
```

Comment:
It may be worth adding an asterisk here to indicate that travel is not covered by default, although foundation members can ask for travel assistance. @mikeal would know the details on how this process works.